src/ctia/auth.clj

```diff
@@ -34,9 +39,14 @@
 
 (def denied-identity-singleton (->DeniedIdentity))
 
-(s/defn ident->map :- (s/maybe {:login (s/maybe s/Str)
-                                :groups (s/maybe [s/Str])})
+(s/defschema IdentityMap
+  {:client-id (s/maybe s/Str)
+   :login (s/maybe s/Str)
+   :groups (s/maybe [s/Str])})
+
```

Comment:
Why are these nullable? In what cases we'd have the keys but not the values?

Reply:
- `client-id` is only for the jwt authentication.
- for the others, it deserves more investigation in the code, I just preserved the output schema of `ident->map`.